ml-proto/test/memory.wast

```diff
@@ -95,7 +95,7 @@
     (loop
       (if
         (i32.eq (get_local 0) (i32.const 0))
-        (br 1)
+        (br 2)
```

Comment:
Sorry for this, I messed up the branching, and accidentally merged the `if` changes into the `return` branch instead of master. This line is part of the `if` changes (`if` now has an implicit block).